content/workshops/ansible_tower/setup.adoc

```diff
@@ -19,12 +19,28 @@ layout: lab
 
 = Workshop Lab Setup
 
-
+image::tower.002.png['Tower']
 
 [IMPORTANT]
-Today's workshop infrastructure is being run in Amazon EC2. +
-Each student will have two instances to use. +
-Each of you should have received an email.  It has the following information on it. +
+- Today's workshop infrastructure is being run in Amazon EC2.
+
+- Each student will have two instances to use.
+
+=== Tower
```

Comment:
Take a look at the build errors in Travis CI
```
ERROR 2017/06/01 03:25:23 asciidoctor: WARNING: workshops/ansible_tower/setup.adoc: line 22: section title out of sequence: expected level 1, got level 2
ERROR 2017/06/01 03:25:23 asciidoctor: WARNING: workshops/ansible_tower/setup.adoc: line 29: section title out of sequence: expected level 1, got level 2
```

I think this is where it's referring. You are missing a 2nd level header.

Take a look at your output of `hugo server` locally and see if you can get those to go away. I'm happy to help if you have any issues.